dlt/pipeline/__init__.py

```diff
@@ -72,6 +72,8 @@ def pipeline(
         dev_mode (bool, optional): When set to True, each instance of the pipeline with the `pipeline_name` starts from scratch when run and loads the data to a separate dataset.
         The datasets are identified by `dataset_name_` + datetime suffix. Use this setting whenever you experiment with your data to be sure you start fresh on each run. Defaults to False.
 
+        refresh (str | TRefreshMode): One of `drop_dataset`, `drop_tables` or `drop_data`. Set this to fully or partially delete and reset the schema, state and destination dataset when running the pipeline.
```

Comment:
please be more specific :)
- drop dataset: - drops all tables for any source that will processed in `run` or `extract` methods. also drops the source state. also warn that schema history is erased
- drop_tables - in similar manner. . also warn that schema history is erased
user should know what will happen when given options is specified